src/node/routes/login.ts

```diff
@@ -111,6 +112,7 @@ router.post("/", async (req, res) => {
 
     throw new Error("Incorrect password")
   } catch (error) {
-    res.send(await getRoot(req, error))
+    const htmlToRender = await getRoot(req, error)
```

Comment:
Very minor nit, but this is `renderedHtml`, right?

Reply:
lol technically I think you're right. i'll change it